spinalcordtoolbox/registration/register.py

```diff
@@ -329,6 +335,220 @@ def register_step_label(src, dest, step, verbose=1):
     return warp_forward_out, warp_inverse_out
 
 
+def register_step_dl_multimodal_cascaded_reg(src, dest, step, verbose=1):
+    """
+    """
+    # Preprocessing steps - TODO maybe not include these here so there is an option for the user to test different ones
+    logger.info(f"\nPreprocess data by setting an isotropic resolution of 1mm "
+                f"and bringing the source image into same space as moving image...")
+    # Isotropic resolution
+    dest_iso_res = image.add_suffix(dest, '_iso_res')
+    run_proc(['sct_resample', '-i', dest, '-o', dest_iso_res, '-mm', '1x1x1'])
```

Comment:
I would recommend replacing any `run_proc` calls with calls to `main()`. For example:

```suggestion
    sct_resample.main(['-i', dest, '-o', dest_iso_res, '-mm', '1x1x1'])
```

You can import `sct_resample` using:

```python
from spinalcordtoolbox.scripts import sct_resample
```